### PR TITLE
BaSP-MR-165:auth trainers

### PR DESCRIPTION
**Pull request by Eugeniagassman**
i made the authentication for trainers

### CODE REVIEW

File: src/routes/activity.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import express from 'express';
 import activitiesController from '../controllers/activity';
 import validations from '../validations/activity';
-/* import verifyToken from '../middleware/authMiddleware'; */
+// import verifyToken from '../middleware/authMiddleware';
```
Comment (Tilajc) [inline src/routes/activity.js:4]: you can delete this comment
```suggestion

```

**Review by Tilajc — APPROVED** [review]
Pretty good job!

**Comment by LucianoStradiot** [comment]
[[FEATURE] - Auth Trainers](https://trello.com/c/fK3ehCrN/165-feature-auth-trainers)

**Comment by LautaroVercellini** [comment]
Works Good Euge!

**Pull request merged**